Packs/QutteraWebsiteMalwareScanner/Integrations/QutteraWebsiteMalwareScanner/QutteraWebsiteMalwareScanner.py

```diff
@@ -0,0 +1,179 @@
+    demisto.debug(f"Start output {start}")
+    readable_output = f'Started scan of {domain}'
+    if start.get('errorstr') != 'success':
+        readable_output = f'Failed to scan domain {domain}'
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.Start",
+        outputs_key_field="error",
+        outputs=start
+    )
+
+def rescan_status(client: Client, args: dict) -> CommandResults:
+    """
+    Retrieves scan status for previously submitted domain
+    name or URL.
+
+    """
+    domain = args["domain"]
+    if not domain:
+        raise ValueError('domain is missing')
+
+    rescan = client._api_request(domain=domain, request_type="PUT", operation="rescan")
+    readable_output = f'Rescan of {domain}'
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.Status",
+        outputs_key_field="error",
+        outputs=rescan
+    )
+
+def scan_report(client: Client, args: dict) -> CommandResults:
+    """
+    Retrieves scan report for previously submitted domain
+    or URL in context or as a File (default)
+
+    """
+    domain = args["domain"]
+    if not domain:
+        raise ValueError('domain is missing')
+
+    report = client._api_request(domain=domain, request_type="GET", operation="statscan")
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.report",
+        outputs_key_field="error",
+        outputs=report
+    )
+
+def report_malware(client: Client, args: dict) -> CommandResults:
+    """
+    Returns malware status for given domain
+
+    """
+    domain = args["domain"]
+    if not domain:
+        raise ValueError('domain is missing')
+
+    malware = client._api_request(domain=domain, request_type="GET", operation="malware")
+
+    return CommandResults(
+        outputs_prefix="QutteraWebsiteMalwareScanning.malware",
+        outputs_key_field="error",
+        outputs=malware
+    )
+
+def status_blacklist(client: Client, args: dict) -> CommandResults:
+    """
+    Returns url blacklisting information and reputation.
+
+    """
+    domain = args["domain"]
+    if not domain:
+        raise ValueError('domain is missing')
+
+    start = client._api_request(domain=domain, request_type="GET", operation="blacklist")
+    readable_output = f"Blacklisted URI's of {domain}"
+    if start.get('errorstr') != 'success':
+        readable_output = f'Failed to get the report for domain {domain}'
+
+    result = CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="",
+        outputs_key_field='QutteraWebsiteMalwareScanning.domain',
+        outputs=start
+    )
+    return_results(result)
+
+
+''' MAIN FUNCTION '''
+def main(params: dict, args: dict, command: str):
+    """main function, parses params and runs command functions
+    """
+    demisto.debug(f"Command called {command}")
+    demisto.results(f"Args are {args}")
+    demisto.results(f"params are {params}")
+    client = Client(params)
```

Comment:
Let's also add a test-module command, 
test-module should test a simple argument less request - just to ensure there's a connection.

Reply:
Fixed